src/main/java/org/folio/bulkops/service/ElectronicAccessService.java

```diff
@@ -28,17 +29,33 @@ public class ElectronicAccessService {
 
 
   public String electronicAccessToString(ElectronicAccess access) {
-    return String.join(SPECIAL_ARRAY_DELIMITER,
+    return electronicAccessToString(access, SPECIAL_ARRAY_DELIMITER);
+  }
+
+  public String itemElectronicAccessToString(ElectronicAccess access) {
+    return electronicAccessToString(access, ARRAY_DELIMITER);
+  }
+
+  public ElectronicAccess restoreElectronicAccessItem(String s) {
+    return restoreElectronicAccessItem(s, SPECIAL_ARRAY_DELIMITER);
+  }
+
+  public ElectronicAccess restoreItemElectronicAccessItem(String s) {
+    return restoreElectronicAccessItem(s, ARRAY_DELIMITER);
+  }
+
+  private String electronicAccessToString(ElectronicAccess access, String delimiter) {
+    return String.join(delimiter,
       isEmpty(access.getRelationshipId()) ? EMPTY : electronicAccessReferenceService.getRelationshipNameById(access.getRelationshipId()),
       isNull(access.getUri()) ? EMPTY : access.getUri(),
       isEmpty(access.getLinkText()) ? EMPTY : access.getLinkText(),
       isEmpty(access.getMaterialsSpecification()) ? EMPTY : access.getMaterialsSpecification(),
       isEmpty(access.getPublicNote()) ? EMPTY : access.getPublicNote());
   }
 
-  public ElectronicAccess restoreElectronicAccessItem(String s) {
+  private ElectronicAccess restoreElectronicAccessItem(String s, String delimiter) {
```

Comment:
Please, rename argument s to smth readable, at least - value.

Reply:
Such naming was in original method .. ok ... rename previous arguments also  or only my method ?

Reply:
Just rename argument s to make code more readable. Thank you!